modules/tinymce/src/core/main/ts/fmt/FormatUtils.ts

```diff
@@ -198,6 +198,15 @@ const areSimilarFormats = (editor: Editor, formatName: string, otherFormatName:
   });
 };
 
+const isSelectorFormat = (format: Format): format is SelectorFormat =>
```

Comment:
Should these be taking `ApplyFormat` as arguments? Or maybe add an overload so that if the input is `ApplyFormat` it'll tell you if the output is a `SelectorFormat` but if the input is `RemoveFormat` it'll tell you if the output is a `RemoveSelectorFormat`

Reply:
Good catch, these really should be `ApplyFormat`.

Reply:
Fixed